Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -98,6 +105,8 @@ export default class ReportsView extends Vue {
     }
 
     private mounted() {
+        this.getPatientData({ hdid: this.user.hdid });
```

Comment:
I think this can be loaded on create

Reply:
moved data loading from mounted to created